README.md

```diff
@@ -246,10 +246,13 @@ You can add a virtual host entry (`server` block) by placing a .conf file in the
     }
 
     # Dockerfile:
-    RUN rm /etc/nginx/sites-enabled/default
-    ADD webapp.conf /etc/nginx/sites-enabled/webapp.conf
-    RUN mkdir /home/app/webapp
-    RUN ...commands to place your web app in /home/app/webapp...
+    ADD webapp.conf /etc/nginx/sites-enabled/default
```

Comment:
Converge `RUN rm ...` and `ADD webapp.conf` to a single instruction to avoid two layers.